wg-TEMPLATE/wg-charter-template.md

```diff
@@ -0,0 +1,24 @@
+# WG YOURWG Charter
+
+## Scope
+
+Include a 2-3 sentence summary of the problem this group is trying to solve.
+Imagine trying to explain your work to a colleague who is familiar with KubeVirt
+but not necessarily all of the internals.
+
+## Governance
+
+### Stakeholder SIGs
+
+A list of SIGs that are relevant to the WG.
+The template may also mention
+* which SIGs are more relevant than others,
+* which subprojects are related to the WG
+
+### Roles and responsibilities
+
+#### Chairs
+- @your-github-handle
```

Comment:
Did we want two names for Chair and Leads?

Reply:
In K8s terms sigs and wgs have chairs, while subprojects have leads - or did I misread that?

Reply:
> Did we want two names for Chair and Leads?

Ah, now I get it - I had suggested that we want two, since otherwise there'd be an interval of vacancy during PTO.

Reply:
Actually I am not really sure whether we even need the lead role inside this template.